src/sequelize/config/config.js

```diff
@@ -5,13 +5,31 @@ module.exports = {
   development: {
     url: process.env.DB_CONNECTION,
     dialect: "postgres",
+    dialectOptions: {
+      ssl: {
+        require: true,
+        rejectUnauthorized: false,
+      },
+    },
   },
   test: {
     url: process.env.DB_CONNECTION,
```

Comment:
don't production database in testing, learn how you can use cross-env to be able to set current environment when testing and if its in test mode switch database and have other database for testing
example:
![image](https://github.com/atlp-rwanda/eagles-ec-be/assets/71966667/cdfa40ea-e1a5-4169-a24b-5c54530816d4)